Installation/cmake/modules/FindLASLIB.cmake

```diff
@@ -0,0 +1,35 @@
+# - Try to find LASLIB
+# Once done this will define
+#
+#  LASLIB_FOUND = LASLIB_FOUND - TRUE
+#  LASLIB_INCLUDE_DIR - include directory for LASlib
+#  LASZIP_INCLUDE_DIR - include directory for LASzip
+#  LASLIB_LIBRARIES   - the libraries (as targets)
+
+# first look in user defined locations
+find_path(LASLIB_INCLUDE_DIR
+          NAMES lasreader.hpp
+         PATHS /usr/local/include/LASlib/ 
+                   ENV LASLIB_INC_DIR
+         )
+         
+find_path(LASZIP_INCLUDE_DIR
+          NAMES mydefs.hpp
+          PATHS /usr/local/include/LASzip/
+                ${LASLIB_INCLUDE_DIR}/../../LASzip/src
+         )
+
+find_library(LASLIB_LIBRARIES
+             NAMES las
+             PATHS ENV LD_LIBRARY_PATH
+                   ENV LIBRARY_PATH
+                   /usr/local/lib
+                   ${LASLIB_INCLUDE_DIR}/../../lib
+                  ENV LASLIB_LIB_DIR
+            )
+
+if(LASLIB_LIBRARIES)
```

Comment:
If `LASLIB_LIBRARIES` evaluates to `TRUE`, but not `LASZIP_INCLUDE_DIR`, there is a CMake error.

On my machine, there is a file `/usr/lib64/liblas.so`, from the package `liblas-devel-1.8.0-13.fc25.x86_64`, that corresponds to http://www.liblas.org, and `LASLIB_LIBRARIES` is automatically set to `/usr/lib64/liblas.so`, but `LASZIP_INCLUDE_DIR` remains `NOTFOUND`.